### PR TITLE
Updates on security sections with links to relevant standards

### PR DESCRIPTION
**Pull request by mprorock**
***
<a href="https://pr-preview.s3.amazonaws.com/mesur-io/did-imp-guide/pull/14.html" title="Last updated on Aug 13, 2021, 1:10 PM UTC (e357420)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-imp-guide/14/ddb3584...mesur-io:e357420.html" title="Last updated on Aug 13, 2021, 1:10 PM UTC (e357420)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -970,18 +980,30 @@ <h3>Vendor Lock</h3>
...
+        lead to greater security, but promote interoprability and adoption.
+      </p>
     </section>
 
     <section class="informative">
       <h3>Digital Signatures</h3>
 
       <p class="advisement">
-        Review
+        We reccomend the user review
```
Comment (OR13) [inline index.html:994]:
```suggestion
        We recommend the user review
```

**Review by OR13 — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -979,18 +989,30 @@ <h3>Vendor Lock</h3>
         Hyperledger Fabric. Design your method such that it may be adapted
         to support multiple ledger systems.
       </p>
+
+      <p class="advisement">
+        Transparency and openess in approaches related to security not only
```
Comment (clehner) [inline index.html:994]:
```suggestion
        Transparency and openness in approaches related to security not only
```

File: index.html [hunk 3]
```diff
@@ -1002,7 +1024,28 @@ <h3>Digital Signatures</h3>
     <section class="informative">
       <h3>Hashing Algorithms</h3>
 
-      <p>Avoid MD5.</p>
+      <p>Avoid MD5, SHA1, and other legacy hashing algorithms with known weeknesses
```
Comment (clehner) [inline index.html:1027]:
```suggestion
      <p>Avoid MD5, SHA1, and other legacy hashing algorithms with known weaknesses
```

**Comment by OR13** [comment]
ping @msporny @peacekeeper

**Review by peacekeeper — APPROVED** [review]
I'm not really a security expert, but looks good to me.

File: index.html [hunk 4]
```diff
@@ -926,6 +933,8 @@ <h3>Compliance</h3>
       <p>Consider <a href="https://gdpr.eu/">GDPR.</a></p>
 
       <p>Consider <a href="https://oag.ca.gov/privacy/ccpa">CCPA.</a></p>
+
+      <p>Consider <a href="https://www.bis.doc.gov/index.php/policy-guidance/encryption">EAR</a></p>
```
Comment (davidlehn) [inline index.html:937]:
```suggestion
      <p>Consider <a href="https://gdpr.eu/">GDPR</a>.</p>

      <p>Consider <a href="https://oag.ca.gov/privacy/ccpa">CCPA</a>.</p>

      <p>Consider <a href="https://www.bis.doc.gov/index.php/policy-guidance/encryption">EAR</a>.</p>
```

**Comment by OR13** [comment]
I'm merging and preparing  an FPWD as directed by the chairs.

**Pull request merged**